aiohttp/client.py

```diff
@@ -420,7 +425,7 @@ async def _request(
         except ValueError as e:
             raise InvalidUrlClientError(str_or_url) from e
 
-        if url.scheme not in HTTP_SCHEMA_SET:
+        if url.scheme not in HTTP_SCHEMA_SET and url.scheme not in WS_SCHEMA_SET:
```

Comment:
It would be slightly faster to have a combined constant, maybe `ALLOWED_URL_SCHEMAS_SET` so there is only one check